.github/workflows/zwave-js-bot.yml

```diff
@@ -0,0 +1,120 @@
+name: "Z-Wave Bot: React to comments"
+
+on:
+  issue_comment:
+    types: [created] # edited, deleted
+
+jobs:
+  # #########################################################################
+  # Fix lint errors when an authorized person posts "@zwave-js-bot fix lint"
+  fix-lint:
+    if: |
+      contains(github.event.issue.html_url, '/pull/') &&
+      contains(github.event.comment.body, '@zwave-js-bot fix lint') &&
+      (github.event.comment.user.login != 'zwave-js-bot')
```

Comment:
oh... might want to exclude `zwave-js-assistant[bot]` aswell.

Reply:
https://github.com/zwave-js/node-zwave-js/commit/2fea442ce0485079f9d0f241c68056ef90a0324f

Reply:
PR opened thanks